### PR TITLE
Feature/km/add recorder timer

### PR DESCRIPTION
**Pull request by resource11**
# Code-review checklist #

  * Documentation
      * [ ] Interfaces describe how other programmers would use them.
      * [ ] Code that is not obvious has documentation describing why
        that code is the way it is.

### CODE REVIEW

File: client/js/_player/components/common/recorder_timer.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+      });
+    } else {
+      this.setState(prevState => ({
+        secsCt       : prevState.secsCt + 1,
+        secsStringCt : prevState.secsCt + 1,
+      }));
+    }
+  }
+
+  handleTimerCount = () => {
```
Comment (BenHSmith13) [inline client/js/_player/components/common/recorder_timer.jsx]: Why not just declare these as functions on the class instead of declaring them as lambdas?

**Comment by BenHSmith13** [comment]
I moved the player style folders into a subfolder, and that is up on master. It is ignored by hound, but you may have some conflict with the moved files. Sorry

File: client/js/_player/components/common/recorder_timer.jsx (hunk 1 above)
Comment (resource11) [inline client/js/_player/components/common/recorder_timer.jsx]: Is the way I've coded this not a best practice? I certainly welcome tips on JS/React best practices.

Comment (resource11) [inline client/js/_player/components/common/recorder_timer.jsx]: Ah, you mean on handleTimerCount. Will do!

**Comment by resource11** [comment]
OK, updates made @BenHSmith13. Can you review again?

File: client/js/_player/components/common/recorder_timer.spec.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+describe('recorder timer', () => {
+
+  let result;
+  let props;
+  beforeEach(() => {
+    props = {
+      timeout: 100,
+    };
+    result = TestUtils.renderIntoDocument(<RecorderTimer {...props} />);
```
Comment (BenHSmith13) [inline client/js/_player/components/common/recorder_timer.spec.js]: You will want to rename this to recorder_timer.spec.jsx for the jsx syntax

**Comment by BenHSmith13** [comment]
You will want to rename this to recorder_timer.spec.jsx for the jsx syntax

**Comment by BenHSmith13** [comment]
Also, the stuff Justin added that fixed the css caused the specs to break. I fixed it in the PR I just opened

**Pull request closed without merging**